docs/sources/clients/promtail/installation.md

```diff
@@ -133,45 +185,3 @@ roleRef:
     name: promtail-clusterrole
     apiGroup: rbac.authorization.k8s.io
 ```
-
-### Sidecar
```

Comment:
Why are we completely removing the sidecar example, just because of the added complexity it adds? I do think this is still worth while to document, if we don't want it here could we consider moving it somewhere else?

Reply:
@trevorwhitney you're right. My thought process was that it didn't help a getting started experience but it is useful to have documented. It should stay in the install doc - it's the best place for it as it's the most relevant place compared to other pages.

Reply:
@trevorwhitney following that one - the current sidecar example isn't a sidecar - it just deploys promtail as a deployment. I suggest we actually do remove this for now and if folks want to deploy it as a side car they can infer how to do that using the above examples. I feel that is safer than providing config that doesn't actually do what we say it does.

What do you think?

Reply:
Sounds good!